lldb/packages/Python/lldbsuite/test/lldbutil.py

```diff
@@ -1654,6 +1654,26 @@ def find_library_callable(test):
     )
 
 
+def target_install(test, filename=None, path=None):
+    test.assertTrue(filename or path, "filename or path must be specified.")
+    if filename is None:
+        filename = os.path.basename(path)
+    if path is None:
+        path = test.getBuildArtifact(filename)
```

Comment:
This seems a bit... magical. Can we just have the caller specify a full path to the thing it wants to install? `self.getBuildArtifact` is not that much extra typing.

Reply:
I expected the following usage:

1: Use filename
```
target_path = lldbutil.install_to_target(self, "a.out")
```
2: Use path
```
local_path = self.getBuildArtifact("some_file")
# write to local_path here 
...
# avoid typing "some_file" constant twice, just extract the filename from the local_path
target_path = lldbutil.install_to_target(self, path=local_path) 
```
3: Use both
```
# "temp" is the filename on the remote target
target_path = lldbutil.install_to_target(self, "temp", local_temp_path_outside_build_dir)
```

Reply:
Ok. I kept only the full path parameter. Updated. Thanks.